model_signing/README.md

```diff
@@ -167,17 +167,31 @@ Machine M1: Debian 6.3.11 x86_64 GNU/Linux, 200GB RAM, 48 vCPUs, 512KB cache, AM
 
 Machine M2: Debian 5.10.1 x86_64 GNU/Linux, 4GB RAM, 2 vCPUs, 56320 KB, Intel(R) Xeon(R) CPU @ 2.20GHz.
 
+H1: Hashing using a tree representation of the directory.
+
+H2: Hashing using a list representation of the directory. (Implementation is parallized with shards of 1GB sizes across vCPUs).
+
 A single run was performed.
 
-| Machine | Model   |      Size      |  Sign Time | Verify Time | 
-|--------|----------|:-------------:|:------:|:------:|
-| M1 | roberta-base-11      | 8K    | 0.8s  | 0.6s  |
-| M1 | hustvl/YOLOP         | 215M  | 1.2s  | 0.8s  |
-| M1 | bertseq2seq          | 2.8G  | 4.6s  | 4.4s  |
-| M1 | bert-base-uncased    | 3.3G  | 5s    | 4.7s  |
-| M1 | tiiuae/falcon-7b     | 14GB  | 12.2s | 11.8s |
-| M2 | roberta-base-11      | 8K    | 1.1s  | 0.7s  |
-| M2 | hustvl/YOLOP         | 215M  | 1.9s  | 1.7s  |
-| M2 | bertseq2seq          | 2.8G  | 18s   | 23.2s |
-| M2 | bert-base-uncased    | 3.3G  | 23.4s | 18.9s |
-| M2 | tiiuae/falcon-7b     | 14GB  | 2m4s | 2m2s   |
+| Hash | Machine | Model   |      Size      |  Sign Time | Verify Time | 
```

Comment:
I think there is a better way to present these benchmarks :).  Perhaps two separate tables, one per machine, or even four separate tables?  Might be worth keeping the old benchmarks too, just for comparison?

Reply:
split into 2 (machine) tables. Thanks